src/components/UriViewer.js

```diff
@@ -292,9 +297,9 @@ export const UriViewerLink = ({ uri, workspace }) => {
       href: uri,
       onClick: e => {
         e.preventDefault()
-        setModalOpen(true)
+        setModalOpen(!isAzureUri(uri))
```

Comment:
? For Azure URLs, why do we render this link if it can't be used?

Reply:
I still think the link should do something. This would appear broken to a user. Until AJ-871, can we open the modal but only display limited information in the modal?

Reply:
I think clicking the link and having a modal with limited information sounds like a better user experience. I suggest the modal say "Download functionality is coming soon."

Reply:
Thanks both -- will update here accordingly then!

Reply:
<img width="1124" alt="Screenshot 2023-02-17 at 11 51 00 AM" src="https://user-images.githubusercontent.com/36093535/219714904-510d5dca-7d5d-4358-920e-a20be851deff.png">

I've updated the code to provide a modal that looks as such -- let me know if this works!

Reply:
note that the logic for what gets a link here is not exactly the same as what WDS uses for its `File` datatype - specifically support for dos:// uris. If we truly wanted these to be in sync we should probably ask WDS for its datatype and display a link based on the column's WDS definition, instead of inspecting the content directly. That can/should be for a future PR.